packages/config/src/main.ts

```diff
@@ -72,10 +72,15 @@ export const resolveConfig = async function (opts) {
     mode,
     offline,
     siteFeatureFlagPrefix,
-    featureFlags,
+    featureFlags: passedFeatureFlags,
     testOpts,
   })
 
+  // Feature flags can also be loaded from the call to `getSiteInfo()`, in the CLI
+  // these won't be accessible within the calls to `netlify-config` as they're fetched
+  // too late. We merge them here to ensure they're available to this package.
+  const featureFlags = { ...passedFeatureFlags, ...(siteInfo.feature_flags ?? {}) }
```

Comment:
I don't think you need this fallback?

```suggestion
  const featureFlags = { ...passedFeatureFlags, ...siteInfo.feature_flags }
```